graylog2-server/src/main/java/org/graylog2/migrations/V20190705071400_AddEventIndexSetsMigration.java

```diff
@@ -206,4 +223,27 @@ private void createEventsStream(String streamId, String streamTitle, String stre
             LOG.error("Couldn't create events stream <{}/{}>! This is a bug!", streamId, streamTitle, e);
         }
     }
+
+    private void ensureSystemNotificationEventsDefinition() {
+        if (dbService.getSystemEventDefinitions().isEmpty()) {
+            EventDefinitionDto eventDto =
+                    EventDefinitionDto.builder()
+                            .title("System notification events")
+                            .description("Reserved event definition for system notification events")
+                            .alert(false)
+                            .priority(1)
+                            .keySpec(ImmutableList.of())
+                            .notificationSettings(EventNotificationSettings.builder()
+                                    .gracePeriodMs(0) // Defaults to 0 in the UI
+                                    .backlogSize(0) // Defaults to 0 in the UI
+                                    .build())
+                            // Empty notifications list by default. The user will specify later in the UI.
+                            .notifications(ImmutableList.<EventNotificationHandler.Config>builder().build())
```

Comment:
do we need that? it should be empty by default
```suggestion

```

Reply:
done